rust/prepare_db.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
❓ I've seen that in itchysats, but what is this used for? the migration will be run automatically in the code.